src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -36,21 +33,32 @@ interface Props {
 function DomainSlider(props: Props): ReactElement {
   const { dataDomain, customDomain, disabled, onCustomDomainChange } = props;
 
-  const [appliedDomain, setAppliedDomain] = useAppliedDomain(
-    dataDomain,
-    customDomain
-  );
-
-  const [tooltipOpen, toggleTooltip] = useToggle(false);
-  useKey('Escape', () => toggleTooltip(false));
+  const appliedDomain = useAppliedDomain(dataDomain, customDomain);
```

Comment:
Note that I now have an `appliedDomain` and a `sliderDomain`. It's the latter that I update in real time when a thumb is moving. Not really necessary yet, but still clearer. It will be more useful later, in my next PR.

Reply:
I get that it is bound to change but can you add a small comment explaining the difference between `sliderDomain` and `appliedDomain` ?

It will be a useful memo (at least for me!)

Reply:
I could also rename this `visDomain` and `useVisDomain` 🤷

Reply:
But then will it make sense in the mapped visualization components? Hmm yeah, I think so. What do you think?